src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -59,6 +60,11 @@ public class ConnectionService
         private readonly ConcurrentDictionary<CancelTokenKey, CancellationTokenSource> cancelTupleToCancellationTokenSourceMap =
                     new ConcurrentDictionary<CancelTokenKey, CancellationTokenSource>();
 
+        /// <summary>
+        /// A map containing the uris of editors whose azure tokens are being refreshed, these editors should have intellisense

```

Comment:
probably the comment should say connections with expired tokens, since we don't know if they are actually being refreshed, or if the refreshed failed/aborted/etc.